app/src/main/java/org/oppia/android/app/profileprogress/ProfileProgressFragmentPresenter.kt

```diff
@@ -20,12 +20,11 @@ private const val TAG_PROFILE_PICTURE_EDIT_DIALOG = "PROFILE_PICTURE_EDIT_DIALOG
 @FragmentScope
 class ProfileProgressFragmentPresenter @Inject constructor(
   private val activity: AppCompatActivity,
-  private val fragment: Fragment
+  private val fragment: Fragment,
+  private val viewModel: ProfileProgressViewModel,
+  private val multiTypeAdapterBuilder: BindableAdapter.MultiTypeBuilder.Factory
```

Comment:
For consistency: `private val multiTypeBuilderFactory: `

Reply:
Updated.